hs_core/views/utils.py

```diff
@@ -888,6 +888,7 @@ def rename_irods_file_or_folder_in_django(resource, src_name, tgt_name):
         # src_name and tgt_name are folder names
         res_file_objs = ResourceFile.list_folder(resource=resource, folder=src_name)
         resource_is_federated = resource.is_federated
+        batch_size = 10-000
```

Comment:
this is how `batch_size` is specified?
https://docs.djangoproject.com/en/3.2/ref/models/querysets/#bulk-update

Reply:
nice, sounds good